xml/System.IdentityModel.Tokens/Saml2AssertionKeyIdentifierClause.xml

```diff
@@ -24,7 +24,9 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Represents a <see cref="T:System.IdentityModel.Tokens.SecurityKeyIdentifierClause" /> implementation for referencing SAML2-based security tokens.</summary>
+    <summary>
```

Comment:
Appears to be coming from https://www.nuget.org/packages/System.ServiceModel.Federation, but we shouldn't include compiler generated docs for this package.